src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SqlDataRecord.cs

```diff
@@ -377,13 +376,11 @@ public SqlDataRecord(params SqlMetaData[] metaData)
                 _columnSmiMetaData[i] = MetaDataUtilsSmi.SqlMetaDataToSmiExtendedMetaData(_columnMetaData[i]);
             }
 
-            _eventSink = new SmiEventSink_Default();
             #if NETFRAMEWORK
```

Comment:
Since these are a lot closer now, can we simplify it to:

```
_recordBuffer = new MemoryRecordBuffer(_columnSmiMetaData);

#if NETFRAMEWORK
_usesStringStorageForXml = true;
#endif
```

Reply:
Yes - thanks. Corrected and committed!